scripts/rebase-ledger.js

```diff
@@ -0,0 +1,48 @@
+const fs = require("fs");
+const sc = require("../node_modules/sourcecred/dist/server/api.js").sourcecred;
+const isEqual = require("../node_modules/lodash.isequal");
+
+const rawToLedgerLog = (rawString) =>
+  JSON.parse("[" + rawString.trim().split("\n").join(",") + "]");
+
+fs.readFile("./data/ledger.json", "utf8", (err, data) => {
```

Comment:
Would be better to take the file paths as arguments.
This would make changes like what I suggested (copy the data to a temp path) simpler as well.

Reply:
accepted